src/Sulu/Bundle/WebsiteBundle/Controller/DefaultController.php

```diff
@@ -41,44 +40,4 @@ public function indexAction(StructureInterface $structure, $preview = false, $pa
 
         return $response;
     }
-
-    /**
-     * Creates a redirect for configured webspaces.
-     *
-     * @param Request $request
-     *
-     * @return Response
-     *
-     * @deprecated since 1.2 use SuluWebsiteBundle:Redirect:redirectWebspace instead
```

Comment:
Would you already delete this? Would consider this a BC break, and would not do that before 2.0. Are any of these methods containing deprecated Symfony code?